code/modules/clothing/under/chameleon.dm

```diff
@@ -12,11 +12,12 @@
 
 	New()
 		..()
-		for(var/U in subtypesof(/obj/item/clothing/under/color))
+		var/blocked = list(/obj/item/clothing/under/color/random, /obj/item/clothing/under/rank/centcom) // Stops random coloured jumpsuit and undefined centcomm suit appearing in the list.
+		for(var/U in subtypesof(/obj/item/clothing/under/color)-blocked)

```

Comment:
spacing between operatee and operator.

color) - blocked) instead of what it is right now